src/pages/derivx/_start-derivx.js

```diff
@@ -0,0 +1,292 @@
+                    onClick={() => onTabClick('Demo')}
+                >
+                    <StyledText size="var(--text-size-m)" align="center">
+                        {localize('Demo account')}
+                    </StyledText>
+                </TabItem>
+                <TabItem
+                    mobile_padding="12px 24px"
+                    active={tab === 'Real'}
+                    onClick={() => onTabClick('Real')}
+                >
+                    <StyledText size="var(--text-size-m)" align="center">
+                        {localize('Real money account')}
+                    </StyledText>
+                </TabItem>
+            </Flex>
+
+            <Flex max_width="1200px">
+                {tab === 'Demo' ? (
+                    <SideTab parent_tab={tab}>
+                        <SideTab.Panel
+                            label=""
+                            description={
+                                <Localize
+                                    translate_text="Sign in to your Deriv account. If you don’t have one, sign up for free."
+                                />
+                            }
+                            mobile_item_width="36rem"
+                        >
+                            <ImageWrapper>
+                                <QueryImage
+                                    data={data[is_mobile ? 'demo_step1_mobile' : 'demo_step1']}
+                                    alt="demo_step1"
+                                />
+                            </ImageWrapper>
+                        </SideTab.Panel>
+                        <SideTab.Panel
+                            label=""
+                            description={
+                                <Localize translate_text="Add a Deriv X demo account." />
+                            }
+                        >
+                            <ImageWrapper>
+                                <QueryImage
+                                    data={data[is_mobile ? 'demo_step2_mobile' : 'demo_step2']}
+                                    alt="demo_step2"
+                                />
+                            </ImageWrapper>
+                        </SideTab.Panel>
+                        <SideTab.Panel
+                            label=""
+                            description={
+                                <Localize translate_text="Start trading on the mobile app or through your web browser." />
+                            }
+                            item_width="36rem"
+                        >
+                            <ImageWrapper>
+                                <QueryImage
+                                    data={data[is_mobile ? 'demo_step3_mobile' : 'demo_step3']}
+                                    alt="demo_step3"
+                                />
+                            </ImageWrapper>
+                        </SideTab.Panel>
+                    </SideTab>
```

Comment:
Is the structure of this condition the same with the next condition? I think we can simplify this by passing the data based on the `tab` prop